src/otx/core/types/export.py

```diff
@@ -98,18 +98,26 @@ def to_metadata(self) -> dict[tuple[str, str], str]:
             dict[tuple[str, str], str]: It will be directly delivered to
             OpenVINO IR's `rt_info` or ONNX metadata slot.
         """
+        has_labels_ids = hasattr(self.label_info, "labels_id")
```

Comment:
I think `labels_id` should be propagated to the `LabelInfo` root class and become a first-class alongside with label names. That'd allow to use both labels and IDs transparently without spreading `is None` checks withing the whole OTX code. We can't generate artificial IDs if the real ones is not accessible.